app/dashboard/messages/page.tsx

```
@@ -36,79 +41,130 @@ interface Message {
-    try {
-      const response = await fetch(`/api/messages/${conversation.id}`);
-      if (!response.ok) {
-        throw new Error("Failed to fetch messages");
-      }
-      const data = await response.json();
+  const {
+    data: messages = [],
+    isLoading: isLoadingMessages,
+  } = useQuery<Message[]>({
+    queryKey: ["messages", selectedConversation?.id],
+    queryFn: () => fetchMessages(selectedConversation!),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)